bleak/backends/bluezdbus/manager.py

```diff
@@ -804,21 +807,21 @@ async def _wait_condition(
 
         event = asyncio.Event()
 
-        def callback():
-            if (
-                self._properties[device_path][defs.DEVICE_INTERFACE][property_name]
-                == property_value
-            ):
+        def _wait_condition_callback(changed: Dict[str, Any]) -> None:
+            """Callback for when a property changes."""
+            if changed.get(property_name) == property_value:
```

Comment:
I think there could be some potential issues here with `None == False`. As mentioned in another comment, might be simpler to just leave it the way it was without passing the `changed` arg.

Reply:
I changed the incoming to pull from `self_interface.get` so `None` will still work

Reply:
I left it passing the value because I always worry about race conditions when we have to look it up again. Not a problem right now but could be in the future if something gets refactored